contracts/test/LightClient.t.sol

```diff
@@ -1064,10 +1064,11 @@ contract LightClient_HotShotCommUpdatesTest is LightClientCommonTest {
         assertEqBN254(hotShotBlockComm, newState.blockCommRoot);
         assertEq(hotShotBlockHeight, newState.blockHeight);
 
-        // Test for max hotShotBlockHeight stored on contract
+        // Test for max hotShotBlockHeight stored on contract that is available (max
+        // hotshotBlockHeight -1)
         // Get the highest HotShot blockheight recorded
         uint256 numCommitments = lc.getStateHistoryCount();
-        (,, hotShotBlockHeight, hotShotBlockComm) = lc.stateHistoryCommitments(numCommitments - 1);
+        (,, hotShotBlockHeight, hotShotBlockComm) = lc.stateHistoryCommitments(numCommitments - 2);
```

Comment:
Hi there, There's an integer underflow here because at this point there's only one state history commitment that would have been set so `numCommitments` = 1. What was the reason for subtracting 2?

Reply:
I hadn't fully fixed this test yet. It should be fixed in this commit. We don't actually need to subtract by 2, but now the function semantic is different in that it will return 1 index in the list past what it previously did. Because of this I had to adjust the height passed into the next function.